.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       matrix:
         os: [ubuntu-20.04]
         python-version: ['3.8']
-        toxenv: [quality, django32, django40]
+        toxenv: [quality, django32, django40, django42]
```

Comment:
We should also update the coverage check to run against `django42` now to verify if that is also working perfectly or not.